index.d.ts

```diff
@@ -2,7 +2,19 @@ declare module "typy" {
 
 	type SafeFunction = (...args: any[]) => any;
 
-	export default function typy (input: any, objectPath?: string): Typy
+	export default function t (obj: any, nestedKeys?: string): Typy;
+	export function addCustomTypes(validators: {[key: string]: any}): any;
+
+	class Schema {
+		public Number: number;
+		public String: string;
+		public Boolean: boolean;
+		public Null: null;
+		public Undefined: undefined;
+		public Array: any[];
+		/* istanbul ignore next */
+		public Function: () => {};
+	}
```

Comment:
Instead of defining it as a class here, we can create an interface that models the `Schema` object and annotate it to a `Schema` variable. `Schema` is actually a member of class Typy as well but I don't know how to add it as such in the declaration file so the one way that we can do this is to do it like this:
```js
interface SchemaType {
		Number: number;
		String: string;
		Boolean: boolean;
		Null: null;
		Undefined: undefined;
		Array: any[];
		Function: () => void;
}

export const Schema: SchemaType;
```

Reply:
done